compose/dm-store.yml

```diff
@@ -69,6 +69,38 @@ services:
     networks:
       - ccd-network
 
+  ccd-case-document-am-api:
+    image: "${CCD_CASE_DOCUMENT_API_USE_LOCAL-hmctspublic.azurecr.io/}ccd/case_document_am_api:${CCD_CASE_DOCUMENT_API_TAG:-latest}"
+    environment:
+      CCD_DOCUMENT_API_IDAM_KEY: "${IDAM_KEY_CASE_DOCUMENT}"
+      CASE_DOCUMENT_S2S_AUTHORISED_SERVICES: ccd_case_document_am_api,ccd_gw,xui_webapp,ccd_data,bulk_scan_processor
+      IDAM_API_URL: http://idam-api:5000
+      IDAM_OIDC_URL: http://idam-api:5000
+      S2S_URL: http://service-auth-provider-api:8080
+      IDAM_USER_URL: http://idam-api:5000
+      DOCUMENT_STORE_URL: http://dm-store:8080
+      CCD_DATA_STORE_URL: http://ccd-data-store-api:4452
+      CASE_DOCUMENT_SWAGGER_AM_URL: http://ccd-case-document-am-api:4455
+      CASE_DOCUMENT_AM_URL: http://ccd-case-document-am-api:4455
+      REFORM_SERVICE_NAME: ccd-case-document-am-api
+      REFORM_TEAM: ccd
+      REFORM_ENVIRONMENT: local
+      APPINSIGHTS_INSTRUMENTATIONKEY: key
+      DOCUMENT_TTL: 3600000
+      # Uncomment this line to enable JVM debugging and uncomment the port mapping below
+      JAVA_TOOL_OPTIONS: '-XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0 -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005'
+    ports:
+      - 4455:4455
+      # Uncomment this and the JAVA_TOOL_OPTIONS flag for JVM debugging
+      - 5005:5005
```

Comment:
comment out as above

Reply:
👍

Reply:
Fixed in https://github.com/hmcts/ccd-docker/pull/119/commits/889a83335d3eeeb7391dc7c2fb83bd18961410c0